dpnp/backend/kernels/dpnp_krnl_indexing.cpp

```diff
@@ -190,7 +183,8 @@ DPCTLSyclEventRef dpnp_diagonal_c(DPCTLSyclQueueRef q_ref,
 
     if (res_ndim <= 1) {
         for (size_t i = 0; i < static_cast<size_t>(res_shape[res_ndim - 1]);
-             ++i) {
+             ++i)
```

Comment:
guess you are using older `clang-format`, please roll back code style changes here